src/compiler/checker.ts

```diff
@@ -20160,6 +20180,31 @@ namespace ts {
                 (containsTruthyCheck(source, (<BinaryExpression>target).left) || containsTruthyCheck(source, (<BinaryExpression>target).right)));
         }
 
+        function getPropertiesPathName(access: AccessExpression | Identifier | ThisExpression, includeIdentifier = false): __String[] | undefined {

```

Comment:
You don't need this helper function, we have a function called `entityNameToString`.